libs/@guardian/libs/src/index.ts

```diff
@@ -58,3 +58,21 @@ export { storage } from './storage/storage';
 
 export type { Switches } from './switches/@types/Switches';
 export { getSwitches } from './switches/getSwitches';
+
+export type {
+	Callback,
```

Comment:
i think this name is a bit too generic for a library with many other exports - how about something like `OnConsentChangeCallback`?